_posts/2025-02-26-PRISM.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: We published the PRISM framework
+author: reto-weber
+---
+We ([Bertrand Meyer][bm] and [Reto Weber][rw]) have published an extended and revised version of the [theory of programs][top], now called [PRISM][pm]. 
```

Comment:
Maybe rephrase without "we". Simply start with names. Or try passive "was published by". Just a suggestion.